app/assets/stylesheets/application.scss

```diff
@@ -63,8 +63,8 @@
     width: 60px;
   }
 
-  .hvr-grow {
-    display: block;
+  .hvr-bounce-in {
+    display: inline-block;
```

Comment:
Properties should be ordered -webkit-backface-visibility, backface-visibility, box-shadow, display, -moz-osx-font-smoothing, -webkit-transform, transform, -webkit-transition-duration, transition-duration, vertical-align